sdk/content-components/examples/ContentSDKWithReport.html

```diff
@@ -9,17 +9,17 @@
       <div style="display: flex;">
         <div style="flex: 1;">
           <!-- sas-content-tree to display a navigation tree -->
-          <sas-content-tree id="ct" url="YOUR-VIYA-URL"></sas-content-tree>
+          <sas-content-tree id="ct"></sas-content-tree>
         </div>
         <div style="flex: 3;">
           <!-- sas-content-breadcrumb to enable navigating back up the tree -->
-          <sas-content-breadcrumb id="cb" url="YOUR-VIYA-URL"></sas-content-breadcrumb>
+          <sas-content-breadcrumb id="cb"></sas-content-breadcrumb>
             <!-- sas-content-area to display a folder containing reports -->
             <!-- Supported attributes: -->
             <!-- url: the url of the Viya server to connect to -->
```

Comment:
It looks like you have removed URL= below, but not sure if it should still be listed as supported.